src/System.Private.DataContractSerialization/src/System/Runtime/Serialization/ClassDataContract.cs

```diff
@@ -243,30 +226,15 @@ internal bool CreateNewInstanceViaDefaultConstructor(out object obj)
             return true;
         }
 
-#if uapaot
-        [RemovableFeature(ReflectionBasedSerializationFeature.Name)]
```

Comment:
`src\Common\src\System\Runtime\CompilerServices\RemovableFeatureAttribute.cs` can be deleted too

Reply:
I wasn't sure about this as https://github.com/dotnet/corefx/blob/master/src/System.Private.DataContractSerialization/src/System/Runtime/Serialization/XmlDataContract.cs#L383 wasn't under an ifdef. Should I remove that too?

Reply:
I think so.